pkg/kubectl/cmd/util/helpers.go

```diff
@@ -428,3 +429,39 @@ func UpdateObject(info *resource.Info, updateFn func(runtime.Object) error) (run
 
 	return info.Object, nil
 }
+
+// isClusterService checks the resource's label to see if it's a cluster service (add-on)
+func isClusterService(info *resource.Info) (bool, error) {
+	if info.Object == nil {
+		info.Get()
```

Comment:
this step can fail, right? what happens if this fails and info.Object is still nil?